eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-binder/src/main/java/com/azure/spring/sample/eventhubs/binder/EventHubBinderApplication.java

```diff
@@ -22,16 +26,13 @@ public static void main(String[] args) {
         SpringApplication.run(EventHubBinderApplication.class, args);
     }
 
-    // Replace destination with spring.cloud.stream.bindings.consume-in-0.destination
-    // Replace group with spring.cloud.stream.bindings.consume-in-0.group
-    @ServiceActivator(inputChannel = "{destination}.{group}.errors")
-    public void consumerError(Message<?> message) {
-        LOGGER.error("Handling consumer ERROR: " + message);
+    @Bean
+    public Consumer<MessageHandlingException> consumerError() {
+        return exception -> LOGGER.error("Handling consumer ERROR", exception);
     }
 
-    // Replace destination with spring.cloud.stream.bindings.supply-out-0.destination
-    @ServiceActivator(inputChannel = "{destination}.errors")
-    public void producerError(Message<?> message) {
-        LOGGER.error("Handling Producer ERROR: " + message);
+    @ServiceActivator(inputChannel = "errorChannel")
+    public void producerError(ErrorMessage errorMessage) {
+        LOGGER.error("Handling Producer ERROR", errorMessage);
     }
```

Comment:
We can still use the errorChannel?

Reply:
Yes, our IT covers this use-case https://github.com/Azure/azure-sdk-for-java/blob/feature/spring-boot-3/sdk/spring/spring-cloud-azure-integration-tests/src/test/java/com/azure/spring/cloud/integration/tests/eventhubs/binder/EventHubsBinderProduceErrorIT.java#L66